### DISCUSSION TITLE
Allow anonymous Retro cards

### DISCUSSION THREAD

**Discussion by markpalfreeman** [microsoft/vsts-extension-retrospectives#163] [Ideas]
Some Agile guides recommend anonymous feedback in the idea-gathering phase (for questions like "What went well?" "What didn't go well?").

It'd be great if the Azure DevOps extension had a toggle switch during creation of a Retro that allowed the posts on Retro boards/columns to be **anonymous**. In this mode, names would not be displayed on posted cards in order to allow everyone to be more comfortable and honest.

**Comment by dphanson63** [comment]
implemented long ago
